modules/caddyhttp/fileserver/browse_test.go

```diff
@@ -1,15 +1,16 @@
 package fileserver
 
 import (
-	"html/template"
 	"testing"
-
-	"github.com/caddyserver/caddy/v2"
+	"text/template"
 )
 
 func BenchmarkBrowseWriteJSON(b *testing.B) {
 	fsrv := new(FileServer)
-	fsrv.Provision(caddy.Context{})
+	fsrv.Browse = &Browse{
+		TemplateFile: "",
+		template:     template.New("test"),
+	}
```

Comment:
I guess these changes aren't necessary anymore, right?

Reply:
> I guess these changes aren't necessary anymore, right?

Right, BrowseWriteJSON is not using the template, so it is not necessary to create a Browse object for the benchmark test.

I want to point out that the deletions of "fsrv.Provision(caddy.Context{})" in both BenchmarkBrowseWriteJSON & BenchmarkBrowseWriteHTML are necessary:
https://github.com/caddyserver/caddy/blob/45fb7202ac0e606ccb7b4fe95f169424f0a6cabc/modules/caddyhttp/fileserver/browse_test.go#L10-L12
https://github.com/caddyserver/caddy/blob/45fb7202ac0e606ccb7b4fe95f169424f0a6cabc/modules/caddyhttp/fileserver/browse_test.go#L31-L33

These 2 lines of changes actually will fix the existing crashes in the benchmark tests of BenchmarkBrowseWriteJSON & BenchmarkBrowseWriteHTML, because fsrv.Provision(caddy.Context{}) will fail due to null pointer of ctx.cfg:
https://github.com/caddyserver/caddy/blob/45fb7202ac0e606ccb7b4fe95f169424f0a6cabc/modules/caddyhttp/fileserver/staticfiles.go#L106
https://github.com/caddyserver/caddy/blob/45fb7202ac0e606ccb7b4fe95f169424f0a6cabc/context.go#L433